tokio/src/stream/mod.rs

```diff
@@ -819,6 +821,33 @@ pub trait StreamExt: Stream {
     {
         Timeout::new(self, duration)
     }
+    /// Slows down a stream by enforcing a delay between items.
+    /// They will be produced not more often than the specified interval.
```

Comment:
I know you just copy-pasted this, but I think the sentence is awkward.

Reply:
Perhaps remove `/// They will be produced not more often than the specified interval.` would improve?

Reply:
I guess it makes sense.